src/core/utils/copyToClipboard.ts

```diff
@@ -16,10 +36,6 @@ export const copyToClipboard: (value: string, options?: { ignoreComment?: boolea
   try {
     navigator.clipboard.writeText(serializedValue)
   } catch {
-    addToast({
-      severity: 'danger',
-      translateKey: 'text_63a5ba11eb4e7e17ef88e9f0',
-    })
-    throw new Error('Browser running in non secure context')
+    unsecuredCopyToClipboard(serializedValue)
```

Comment:
I think we should still warn users about their unsecure context when calling `unsecuredCopyToClipboard`.
Lago is supposed to run in https context but probably throwing an error here is not necessary now.
maybe we should have 2 copies then?

Probably warning (not danger severity maybe?) about unsecure context and later another error toast not having access to clipboard would be ideal?